Source/WebKit/UIProcess/Launcher/cocoa/ProcessLauncherCocoa.mm

```diff
@@ -128,7 +128,7 @@ static void launchWithExtensionKitFallback(ProcessLauncher& processLauncher, Pro
 
 bool ProcessLauncher::hasExtensionsInAppBundle()
 {
-#if PLATFORM(IOS_SIMULATOR)
+#if PLATFORM(IOS) || PLATFORM(IOS_SIMULATOR)
```

Comment:
```suggestion
#if PLATFORM(IOS)
```

`PLATFORM(IOS)` covers both device and simulator.

Reply:
Good point! I just actioned this feedback, thank you!